Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2829,6 +2829,20 @@ GenericCueAPIEnabled:
     WebCore:
       default: false
 
+GetCoalescedEventsEnabled:
+  type: bool
+  status: unstable
```

Comment:
Up to you really, but testable will make it run in WKTR, which we'll  want eventually.

Reply:
I plan on making it `testable` once I have an actual implementation that does something